docker/docker_image_src.go

```diff
@@ -236,33 +236,41 @@ func (s *dockerImageSource) ensureManifestIsLoaded(ctx context.Context) error {
 	return nil
 }
 
-func (s *dockerImageSource) getExternalBlob(ctx context.Context, urls []string) (io.ReadCloser, int64, error) {
+func (s *dockerImageSource) getExternalBlob(ctx context.Context, urls []string) (_ io.ReadCloser, _ int64, fallback bool, _ error) {
 	var (
 		resp *http.Response
 		err  error
 	)
 	if len(urls) == 0 {
-		return nil, 0, errors.New("internal error: getExternalBlob called with no URLs")
+		return nil, 0, true, errors.New("internal error: getExternalBlob called with no URLs")
```

Comment:
Shouldn't this be `false` since it's a fatal error?

Reply:
Same for the other case.  It would be helpful to add a comment above the function to describe what the (boolean) return values mean.